Fix: FontAwesome doesn't work when optional text for URL is added

shervElmi opened the pull request:
### Issue Link:
https://github.com/Strategy11/business-directory-premium/issues/115

### Reproduce:

- Install FontAwesome on a site with the Premium License
- Choose a table layout under the Appearance tab
- In the URL field, choose "replace the value with icon"
- In the listing, add a URL AND text in the optional text field
- The optional text overrides the icon setting as shown below:

### Before the PR:
![image](https://user-images.githubusercontent.com/69119241/224565324-8d04d02a-86c1-47fd-bb92-4cae32c679e0.png)

### After the PR:
<img width="798" alt="image" src="https://user-images.githubusercontent.com/69119241/224565587-0c9448eb-041c-4624-9044-4a2f32fccc66.png">

garretlaxton reviewed (commented):
This looks good, but if I try to switch to a different icon before saving, the modal just closes when I try to click anything.

shervElmi commented:
> This looks good, but if I try to switch to a different icon before saving, the modal just closes when I try to click anything.

@stephywells I am encountering a similar problem as Garret. Although it is a different issue, would you recommend creating a new ticket, or can I address it in this current one?

shervElmi commented:
Thanks, @garretlaxton!

stephywells commented:
@shervElmi since they are both tiny fixes, I think it's fine to include the other fix here too.

shervElmi commented:
@stephywells I have resolved all the issues and addressed your comments in the PR. I believe it's now ready for merging. Let me know if you have any final feedback before I proceed. Thanks!

stephywells reviewed (approved):
This code here looks good. We can merge this once @garretlaxton signs off (I didn't test it)

garretlaxton reviewed (approved):
This works great! Can now select different icons without an issue.

I also tested on the other 2 listing layouts.
